### PR TITLE
Update Neuronpedia embed to better support query parameters, consistent UI

### PR DESCRIPTION
**Pull request by hijohnnylin**
This is an update to #9500, which added Neuronpedia embed support to LessWrong.

@b0b3rt was correct (https://github.com/ForumMagnum/ForumMagnum/pull/9500#issuecomment-2246359360) that the previous implementation was not flexible enough to support cases where the query parameters did not all exist. This fixes that, and also adds support for non-hardcoded query parameters so that we don't need to make changes to ForumMagnum/LW every time we want to support a new query parameter.

This also updates the UI so that the preview editor matches the post.

I've tested various URL types to ensure that this works locally:
- `https://neuronpedia.org/gpt2-small/0-res-jb/14057?embed=true&embedexplanation=false&embedplots=true&defaulttesttext=the+Jedi`
- `https://neuronpedia.org/gpt2-small/0-res-jb/14057?embed=true&embedexplanation=false`
- `https://neuronpedia.org/gpt2-small/0-res-jb/14057`

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1208752957789808) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/linkPreview/PostLinkPreview.tsx [hunk 1]
```diff
@@ -762,7 +764,7 @@ const NeuronpediaPreview = ({classes, href, id, children}: {
         </a>
 
         <LWPopper open={hover} anchorEl={anchorEl} placement="bottom-start">
-          <iframe className={classes.iframeStyling} src={url} />
+          <iframe className={classes.iframeStyling} scrolling="no" src={url} />
```
Comment (oetherington) [inline packages/lesswrong/components/linkPreview/PostLinkPreview.tsx:767]: FYI `scrolling` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#scrolling)

File: public/lesswrong-editor/src/editorConfigs.ts [hunk 2]
```diff
@@ -59,10 +59,10 @@ const embedConfig = {
 		},
 		{
 			name: "Neuronpedia",
-			url: /^neuronpedia\.org\/([a-zA-Z0-9-/]*)(?:\?embed=true&embedexplanation=([a-zA-Z0-9-/]*)&embedplots=([a-zA-Z0-9-/]*)&embedtest=([a-zA-Z0-9-/]*)&height=([a-zA-Z0-9-/]*))?/,
-			html: ([match, identifier, explanation, plots, test, height]: RegExpMatchArray) => `
+			url: /^neuronpedia\.org\/([a-zA-Z0-9-/]*)(?:\?embed=true(.*))?/,
```
Comment (b0b3rt) [inline public/lesswrong-editor/src/editorConfigs.ts:62]: @hijohnnylin is this now matching to any string after the embed param?  I think I'd prefer a narrower thing; if the intent is to support arbitrary future query params, a repeating match on `&([a-zA-Z0-9-]*)=([a-zA-Z0-9-]*)` (or something like that; I'm not sure that's totally correct) seems safer.

**Review by b0b3rt — APPROVED** [review]
Sorry for the late review, been kind of slammed.  Looks good, thank you!

**Comment by hijohnnylin** [comment]
@b0b3rt thanks! i did the merge from main and rebuild just now.

**Comment by hijohnnylin** [comment]
@b0b3rt just checking in on this - anything else I should be doing?

**Comment by b0b3rt** [comment]
@hijohnnylin nope, sorry, just been both busy and sick 😆

Will merge.

**Comment by hijohnnylin** [comment]
> @hijohnnylin nope, sorry, just been both busy and sick 😆
> 
> Will merge.

ahh so sorry. thank you and hope you are better soon :(

**Pull request merged**